google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2192,7 +2618,8 @@ enum Position {
   private final CountDownLatch waitOnMinSessionsLatch;
   private final SessionReplacementHandler pooledSessionReplacementHandler =
       new PooledSessionReplacementHandler();
-
+  private final SessionReplacementHandler multiplexedSessionReplacementHandler =
```

Comment:
nit: this could even be `private static final`

Reply:
done.